### PR TITLE
[8.0][IMP][account_banking_*] Define SEPA identifiers per payment mode

### PR DESCRIPTION
**Pull request by antespi**
This PR allows to define SEPA identifiers by payment mode.
For companies who have several bank accounts and wants to create payment orders from some of them, have the problem that one bank gives them different identifiers then other bank.
Sometimes, SEPA identifiers are used by bank to identify financial services, so they are different between bank accounts from the same bank.

### CODE REVIEW

File: account_banking_pain_base/models/payment_mode.py [hunk 1]
```diff
@@ -14,3 +15,35 @@ class PaymentMode(models.Model):
...
+        help="This will be used as the 'Initiating Party Identifier' in "
+        "the PAIN files generated by Odoo. If not defined, Initiating Party "
+        "Identifier from company will be used.\n"
+        "Common format (13): \n"
+        "- Country code (2, optional)\n"
+        "- Company idenfier (N, VAT)\n"
+        "- Service suffix (N, issued by bank)")
+    sepa_type = fields.Char(compute="_compute_sepa_type")
+
+    def _sepa_type_get(self):
```
Comment (pedrobaeza) [inline account_banking_pain_base/models/payment_mode.py:38]: Why 2 methods? You can inherit the compute method directly

Comment (antespi) [inline account_banking_pain_base/models/payment_mode.py:38]: Well, for not doing the loop again in all inherited methods. Just get the type.
The api.depends one takes care about multi record and fields dependency

Comment (pedrobaeza) [inline account_banking_pain_base/models/payment_mode.py:38]: OK then, we can let it as is.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by sergio-teruel** [comment]
:+1:

**Comment by rafaelbn** [comment]
Thanks! Longtime waiting por this IMP :+1:

**Comment by carlosdauden** [comment]
:+1:

File: account_banking_sepa_credit_transfer/README.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
     :alt: License: AGPL-3
-    
+
 Account Banking SEPA Credit Transfer
```
Comment (elicoidal) [inline account_banking_sepa_credit_transfer/README.rst:4]: line above

File: account_banking_sepa_credit_transfer/README.rst [hunk 3]
```diff
@@ -62,15 +62,16 @@ Credits
...
+* Stefan Rijnhart
+* Julien Laloux
+* Alexandre Fayolle
+* Raphaël Valyi
+* Erwin van der Ploeg
+* Sandy Carter
+* Antonio Espinosa <antonioea@antiun.com>
 
 Maintainer
 ----------
```
Comment (elicoidal) [inline account_banking_sepa_credit_transfer/README.rst:77]: below: update the url (https x2)

File: account_banking_sepa_direct_debit/README.rst [hunk 4]
```diff
@@ -49,7 +49,7 @@ Known issues / Roadmap
 ======================
```
Comment (elicoidal) [inline account_banking_sepa_direct_debit/README.rst:49]: Can you take the opportunity to correct the README following the newest template?

**Pull request merged**